README.md

```
@@ -9,32 +9,32 @@
 [![Codecov](https://codecov.io/gh/acgetchell/causal-triangulations/graph/badge.svg?token=CsbOJBypGC)](https://codecov.io/gh/acgetchell/causal-triangulations)
 [![Audit dependencies][audit-badge]][audit-workflow]
 
-Causal Dynamical Triangulations for quantum gravity in [Rust], built on fast Delaunay triangulation primitives.
+Causal Dynamical Triangulations for quantum gravity in [Rust], built on fast [Delaunay triangulation] primitives
+and composable, adaptable Metropolis-Hastings sampling via [`markov-chain-monte-carlo`].
+
+## Contents
+
+- [Introduction](#-introduction)
+- [Project Status](#-project-status)
+- [Ensemble And Volume Behavior](#-ensemble-and-volume-behavior)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)